### DISCUSSION TITLE
Virtual Product WooCommerce

### DISCUSSION THREAD

**Discussion by frank927** [woocommerce/woocommerce#50184] [Checkout Flow]
Hi, I am trying to delete the billing address for a virtual product on the checkout page and I have not been able to, I use the HivePress theme, if you could guide me I would greatly appreciate it.

**Comment by IanMisnerThough** [comment]
Not sure this venue is intended for this sort of thing as it's a bit more like support, but what you're looking for is available here: https://www.businessbloomer.com/woocommerce-hide-checkout-billing-fields-if-virtual-product-cart/

That being said, a lot of payment gateways will want you to collect billing details on checkout still.

**Comment by nielslange** [comment]
Hello @frank927 👋,

In this [discussion](https://github.com/woocommerce/woocommerce/discussions/45568#discussioncomment-11037949) on GitHub, @senadir has shared a tutorial which can be found on his blog: [Removing fields from Checkout block](https://nadir.blog/2024/07/22/removing-fields-from-checkout-block/). This guide explains how to remove specific fields from the WooCommerce Checkout block, which might be exactly what you need for your project. I hope this helps you achieve the desired outcome.
